### ISSUE TITLE
Training data is never used

### ISSUE THREAD

**Issue by GuanxiongLiu** [devsisters/pointer-network-tensorflow#9]
Below is line 40 and line 45 in model.py. The is_training variable is always set to False without changing based on config.

`

    self.is_training = tf.placeholder_with_default(
        tf.constant(False, dtype=tf.bool),
        shape=(), name='is_training'
    )
`
`

    self.enc_inputs, self.dec_targets, self.enc_seq_length, self.dec_seq_length, self.mask = 
        smart_cond(
            self.is_training,
            lambda: (inputs['train'], labels['train'], enc_seq_length['train'],
                     dec_seq_length['train'], mask['train']),
            lambda: (inputs['test'], labels['test'], enc_seq_length['test'],
                     dec_seq_length['test'], mask['test'])
        )
`

**Comment by GuanxiongLiu** [comment]
Here is my temp fix in model.py. It could run with training command but I never let it finish.

`

    is_training = False if config.is_train == 'False' else True
    self.is_training = tf.placeholder_with_default(
        tf.constant(is_training, dtype=tf.bool),
        shape=(), name='is_training'
    )
`
